src/main/java/io/github/syst3ms/skriptparser/sections/SecFlatMap.java

```diff
@@ -5,18 +5,34 @@
 import io.github.syst3ms.skriptparser.lang.Expression;
 import io.github.syst3ms.skriptparser.lang.Statement;
 import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.Variable;
+import io.github.syst3ms.skriptparser.lang.control.SelfReferencing;
 import io.github.syst3ms.skriptparser.lang.lambda.ReturnSection;
-import io.github.syst3ms.skriptparser.lang.lambda.SkriptFunction;
 import io.github.syst3ms.skriptparser.log.ErrorType;
 import io.github.syst3ms.skriptparser.log.SkriptLogger;
 import io.github.syst3ms.skriptparser.parsing.ParseContext;
 import io.github.syst3ms.skriptparser.parsing.ParserState;
 import io.github.syst3ms.skriptparser.types.changers.ChangeMode;
+import io.github.syst3ms.skriptparser.util.Pair;
+import org.jetbrains.annotations.Nullable;
 
-import java.util.Arrays;
-import java.util.Optional;
+import java.util.*;
```

Comment:
do not import all java utils (it's your IDE fault i guess)

Reply:
Yup. I'll see if I can disable this behaviour.